SQLite3/module.modulemap

```diff
@@ -0,0 +1,5 @@
+module SQLite3 [system] {
+    header "/usr/include/sqlite3.h"
```

Comment:
According to the svdo/swift-netutils link, yes, this can be made to work just for CocoaPods. I'll give it a try.